### DISCUSSION TITLE
Feature request - pause SLA timer and recalculate "Due date"

### DISCUSSION THREAD

**Discussion by Andruxxxxxa** [osTicket/osTicket#6438] [Ideas]
I kindly request that OS Ticket could include a feature that enables the pause of the SLA timer, allowing for the recalculation of the due date when the timer resumes. Use case is the following - Our customer contract outlines penalties for failing to meet SLA targets; we calculate our "pure" work, not including the time required by the customer to collect and provide the necessary information. Scenario is quite usual for many companies.

I thank you for taking the time to consider this request.

**Comment by daotranid** [comment]
1 vote for you. My company also need it
